app/routes/_protected._index.tsx

```diff
@@ -0,0 +1,129 @@
+import { useOpenSecret } from '@opensecret/react';
+import type { MetaFunction } from '@vercel/remix';
+import { useState } from 'react';
+import { Button } from '~/components/ui/button';
+import { useTheme } from '~/features/theme';
+
+export const meta: MetaFunction = () => {
+  return [
+    { title: 'Boardwalk' },
+    { name: 'description', content: 'Welcome to Boardwalk!' },
+  ];
+};
+
+export default function Index() {
+  const os = useOpenSecret();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [name, setName] = useState('');
+  const { theme, effectiveColorMode, colorMode, setTheme, setColorMode } =
+    useTheme();
+
+  if (os.auth.loading) {
+    return <div>Loading...</div>;
+  }
+
+  if (os.auth.user) {
+    return (
+      <div>
+        <h1>Welcome to Boardwalk!</h1>
+        <div>id: {os.auth.user.user.id}</div>
+        <div>email: {os.auth.user.user.email}</div>
+        <div>name: {os.auth.user.user.name}</div>
+        <div>email verified: {os.auth.user.user.email_verified}</div>
+        <div>login method: {os.auth.user.user.login_method}</div>
+        <div>created at: {os.auth.user.user.created_at}</div>
+        <div>updated at: {os.auth.user.user.updated_at}</div>
+        <Button variant="default" onClick={() => os.signOut()}>
+          Log out
+        </Button>
+      </div>
+    );
+  }
+
+  return (
```

Comment:
Why is this here? Shouldn't the user already be redirected to /login?

Reply:
that's old leftover. I haven't pushed the latest code yet